channel_test.go

```diff
@@ -21,6 +21,23 @@ func (ch *Channel) refresh(ctx context.Context) error {
 	return err
 }
 
+func TestClient_TestQuery(t *testing.T) {
+	c := initClient(t)
+	membersID := randomUsersID(t, c, 1)
+	ch := initChannel(t, c, membersID...)
+	ctx := context.Background()
+	_, err := ch.SendMessage(ctx, &Message{Text: "test message", Pinned: true}, ch.CreatedBy.ID)
+	require.NoError(t, err)
+	defer func() {
```

Comment:
nit: since go1.14, we have [t.Cleanup()](https://pkg.go.dev/testing#T.Cleanup) which does this for you.

Reply:
I'll try to unify it in the future for every test method.